src/mumble/CoreAudio.h

```diff
@@ -8,30 +8,41 @@
 
 #	include "AudioInput.h"
 #	include "AudioOutput.h"
-
 #	include <AudioToolbox/AudioToolbox.h>
-#	include <Carbon/Carbon.h>
 
-#	include "Global.h"
+enum AUDirection { OUTPUT = 0, INPUT = 1 };
```

Comment:
@Krzmbrzl I used enum instead of enum class in the end. That is because: in https://github.com/mumble-voip/mumble/blob/e518dc745e57fd8817cb4774dc5085d419be6b05/src/mumble/CoreAudio.mm#L581-L597, the value of this enum is directly passed to Mac's API. Using enum class will require me to cast them into integers explicitly otherwise the compiler will complain.

I think since this is the only enum defined for CoreAudio backend, an enum is okay for this case.

Reply:
Yeah sure. That's exactly what I meant with saying an enum _class_ might not be suitable when having to cast to and from ints. So all good :+1: